posts/inside-rust/2020-11-02-Using-rustc_cranelift_codegen.md

```diff
@@ -78,7 +77,7 @@ index bccb41121..703afa754 100644
  
  pub const CARGO_ENV: &str = "CARGO";
 EOF
-$ ../rustc_codegen_cranelift/build.sh
+$ ../rustc_codegen_cranelift/build/build.sh
```

Comment:
This is inconsistent with `build.sh` above, which isn't in the `build/` subdirectory. Which did you mean? @bjorn3

Reply:
Ah, this should be `build/cargo.sh build`.